pkgs/by-name/ei/eiffel-studio-bin/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  autoPatchelfHook,
+  fetchzip,
+  gcc,
+  gnumake,
+  gtk2-x11,
+  lib,
+  makeWrapper,
+  stdenv,
+  xorg,
+  build ? "103187",
+  libraries ? [ ],
+  platform ? "linux-x86-64",
+  version ? "19.05",
+}:
+
+stdenv.mkDerivation {
+  pname = "eiffelstudio-bin";
+  version = version;
+  src = fetchzip {
+    url = "mirror://sourceforge/eiffelstudio/EiffelStudio%20${version}/Build_${build}/Eiffel_${version}_gpl_${build}-${platform}.tar.bz2";
+    hash = "sha256-Tt4u4hydiIXB7FQnaoE/GtJt0gYRMOaVdDDUgMrDJtU=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    gtk2-x11
+    makeWrapper
+    xorg.libXtst
```

Comment:
libXtst is a library and goes into buildInputs like gtk2-x11. Also usually gtk2 should be enough